sdk/core/azure-core/azure/core/polling/base_polling.py

```diff
@@ -80,6 +79,30 @@
 _SUCCEEDED = frozenset(["succeeded"])
 
 
+def update_api_version_of_status_link(
+    status_link: str, lro_options: Optional[Dict[str, Any]] = None
+) -> Tuple[str, Dict[str, Any]]:
+    """Handle status link.
+
+    :param status_link: Lro status link.
+    :type status_link: str
+    :param lro_options: Options for LRO.
+    :type lro_options: dict[str, Any]
+    :return: Parsed status link and parsed query parameters.
+    :rtype: Tuple[str, Dict[str, Any]]
+    """
+    request_params: Dict[str, Any] = {}
+    if lro_options and "api_version" in lro_options:
+        parsed_status_link = urllib.parse.urlparse(status_link)
+        request_params = {
+            key.lower(): [urllib.parse.quote(v) for v in value]
```

Comment:
We can't change the casing of query parameters in the status link....

Reply:
Updated